indico/modules/events/registration/clone.py

```diff
@@ -26,65 +28,57 @@
 from indico.modules.events.registration.models.registrations import Registration, RegistrationData
 from indico.util.i18n import _
 
-from MaKaC.conference import EventCloner
-
 
 class RegistrationFormCloner(EventCloner):
-    def __init__(self, *args, **kwargs):
-        super(RegistrationFormCloner, self).__init__(*args, **kwargs)
-        self._field_data_map = {}
-        self._clone_with_registrations = False
-
-    def get_options(self):
-        if not is_feature_enabled(self.event, 'registration'):
-            return {}
-        forms_cloner_enabled = bool(self._find_registration_forms().count())
-        regs_cloner_enabled = bool(self._find_registration_forms()
-                                   .filter(RegistrationForm.registrations.any(Registration.is_active))
-                                   .count())
-        selected = self.event.getType() == 'meeting'
-        return {
-            'form': (_('Registration forms'), forms_cloner_enabled, False),
-            'registrations': (_('Registrations'), regs_cloner_enabled, selected)
-        }
+    name = 'registration_forms'
+    friendly_name = _('Registration forms')
```

Comment:
pff, i never complained about people using single quotes there ;) it's ome doing that!